pkg/apis/observability/v1alpha1/types.go

```diff
@@ -42,11 +42,35 @@ type ClusterObservabilitySpec struct {
 
 	// Capabilities defines the observability capabilities.
 	// Each capability has to be enabled explicitly.
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Capabilities",order=2
 	Capabilities *CapabilitiesSpec `json:"capabilities,omitempty"`
 }
 
 // ClusterObservabilityStatus defines the observed state of ClusterObservability.
-type ClusterObservabilityStatus struct{}
+type ClusterObservabilityStatus struct {
+	// OpenTelemetry defines the status of the OpenTelemetry capability.
+	// +optional
+	OpenTelemetry string `json:"opentelemetry,omitempty"`
```

Comment:
What is the string, why isn't it a Condition?

Reply:
It's ns/instance-name (version) . I will add a comment in a follow up PR